### PR TITLE
Homework dictionary

### PR DESCRIPTION
**Pull request by Pticatos86**
I have a question: How correctly to close block loops in task 1 ? Is my decision in task 1 correct? For task 2 and 3 i have no questions.

### CODE REVIEW

File: dictionary/homework_dictionary.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        else:
+            pass
+"""I think for correcting end current loops should be block else,
+but I don`t know what to write in this block else.
+Is it correct if i write else: pass?"""
+
+if check_name in list_system_names:
+    print(f"Hello {system_role}")
+else:
+    print("Hello guest")
```
Comment (MarinaZelenska) [inline dictionary/homework_dictionary.py]: Can be better
image

File: dictionary/homework_dictionary.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+"""Solution task 2"""
+params = {'v': 'some_id', 'list': 'some_list', 'index': '6', 't': 'Os'}
+initial_str = 'https://www.youtube.com/watch?'
+
+params_list = []
+for key, value in params.items():
+    params_list.append(key + '=' + value)
```
Comment (MarinaZelenska) [inline dictionary/homework_dictionary.py]: You can use string in this case.

File: dictionary/homework_dictionary.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+params_string = '&'.join(params_list)
+print(f"Result link is {initial_str + params_string}")
```
Comment (MarinaZelenska) [inline dictionary/homework_dictionary.py]: Can be better
image

File: dictionary/homework_dictionary.py [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+key_two = 'a'
+value_two = result_link.count(key_two)
+result_dict[key_two] = value_two
+
+key_three = 't'
+value_three = result_link.count(key_three)
+result_dict[key_three] = value_three
+
+print(f"Result dictionary from link {result_link} is {result_dict}")
```
Comment (MarinaZelenska) [inline dictionary/homework_dictionary.py]: Too hard, can be simpler
image

File: dictionary/homework_dictionary.py [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+    params_list.append(key + '=' + value)
+
+params_string = '&'.join(params_list)
+print(f"Result link is {initial_str + params_string}")
+
+"""Solution task 3"""
+result_link = 'https://www.youtube.com/watch?v=some_id&list=some_list&index=6&t=Os'
+result_dict = {}
+
+key_one = 'w'
```
Comment (MarinaZelenska) [inline dictionary/homework_dictionary.py]: You should use loop for decision

File: dictionary/homework_dictionary.py (hunk 1 above)
Comment (Pticatos86) [reply dictionary/homework_dictionary.py]: fixed block loop

File: dictionary/homework_dictionary.py (hunk 3 above)
Comment (Pticatos86) [reply dictionary/homework_dictionary.py]: fixed

File: dictionary/homework_dictionary.py (hunk 4 above)
Comment (Pticatos86) [reply dictionary/homework_dictionary.py]: fixed and analyzed

File: dictionary/homework_dictionary.py (hunk 5 above)
Comment (Pticatos86) [reply dictionary/homework_dictionary.py]: Ok. fixed

File: dictionary/homework_dictionary.py [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+    'maintainer': ['Julia', 'Alex'],
+    'manager': ['Piter', 'Max', 'Sophia'],
+    'developer': ['Natali', 'Nikita', 'Diana']
+}
+
+for role, list_names in roles_dict.items():
+    if check_name in list_names:
+        print(f"Hello {role}")
+        break
+    else:
```
Comment (MarinaZelenska) [inline dictionary/homework_dictionary.py]: block else should be added for loop, not for if. Block else for loops work if block break didn't work

**Review by MarinaZelenska — APPROVED** [review]

**Pull request merged**